src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.ts

```diff
@@ -62,6 +63,32 @@ export class DotTemplateListComponent implements OnInit, OnDestroy {
                 this.templateBulkActions = this.setTemplateBulkActions();
             });
         this.setAddOptions();
+
+        /**
+         * When the portlet reload (from the browser reload button), the site service emits
+         * the switchSite$ because the `currentSite` was undefined and the loads the site, that trigger
+         * an unwanted reload.
+         *
+         * This extra work in the filter is to prevent that extra reload.
+         *
+         */
+        let currentHost = this.dotSiteService.currentSite?.hostname || null;
+
+        this.dotSiteService.switchSite$
+            .pipe(
+                takeUntil(this.destroy$),
+                filter((site: Site) => {
+                    if (currentHost === null) {
+                        currentHost = site.hostname;
+                        return false;
+                    }
+
+                    return true;
+                })
+            )
+            .subscribe(() => {
+                this.dotRouterService.gotoPortlet('templates');
```

Comment:
implements a reload,  looks like this might work, https://stackoverflow.com/a/58114797/3847132